contracts/KetlAttestation.sol

```diff
@@ -61,15 +61,20 @@ pragma solidity ^0.8.19;
 
 import "@big-whale-labs/versioned-contract/contracts/Versioned.sol";
 import "@opengsn/contracts/src/ERC2771Recipient.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
-import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
-import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC1155/ERC1155Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol";
 import "@zk-kit/incremental-merkle-tree.sol/IncrementalBinaryTree.sol";
 import "./interfaces/IAttestationCheckerVerifier.sol";
 import "./interfaces/IPasswordCheckerVerifier.sol";
 
-contract KetlAttestation is ERC1155, Ownable, Versioned, ERC2771Recipient {
-  using Counters for Counters.Counter;
+contract KetlAttestation is
+  ERC1155Upgradeable,
+  OwnableUpgradeable,
+  Versioned,
```

Comment:
Should `Versioned` be upgradeable too? We can ditch the `Versioned` superclass and simply have a `version` variable that is upgraded when deployed

Reply:
`Versioned` is a pretty lightweight class and basically does the same thing. I referenced OBSS, and it uses `Versioned` as an upgradeable contract and sets it like this.

Reply:
Fixed in https://github.com/BigWhaleLabs/ketl-attestation-token/pull/23/commits/34b4f0f205983295d728f27d6813fa4b984abd66.